nipype/interfaces/setup.py

```diff
@@ -24,6 +24,7 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('mne')
     config.add_subpackage('mrtrix')
     config.add_subpackage('mrtrix3')
+    config.add_subpackage('niftyseg')
```

Comment:
Oh there are some issues for the test since most executables in niftyseg don't have a --version options that is working.

I will work on fixing this later today.